### PR TITLE
issue-745: Set timeoutSeconds to 50sec

### PR DESCRIPTION
**Pull request by shunki-fujita**
https://github.com/cybozu-go/moco/issues/745

### CODE REVIEW

File: clustering/operations.go [hunk 1]
```diff
@@ -241,7 +240,7 @@ func (p *managerProcess) failover(ctx context.Context, ss *StatusSet) error {
 
 	gtid := candidates[candidate].ReplicaStatus.RetrievedGtidSet
 	log.Info("waiting for the new primary to execute all retrieved transactions", "index", candidate, "gtid", gtid)
-	err = ss.DBOps[candidate].WaitForGTID(ctx, gtid, failOverTimeoutSeconds)
+	err = ss.DBOps[candidate].WaitForGTID(ctx, gtid, timeoutSeconds)
 	if err != nil {
 		return err
 	}
```
Comment (ymmt2005) [inline clustering/operations.go:246]: I think we can loop this if `err` is `dbop.ErrTimeout`.
https://github.com/cybozu-go/moco/blob/main/pkg/dbop/replication.go#L69

File: clustering/operations.go [hunk 2]
```diff
@@ -163,7 +162,7 @@ func (p *managerProcess) switchover(ctx context.Context, ss *StatusSet) error {
 		return fmt.Errorf("failed to get the primary status: %w", err)
 	}
 
-	err = ss.DBOps[ss.Candidate].WaitForGTID(ctx, pst.GlobalVariables.ExecutedGTID, switchOverTimeoutSeconds)
+	err = ss.DBOps[ss.Candidate].WaitForGTID(ctx, pst.GlobalVariables.ExecutedGTID, timeoutSeconds)
 	if err != nil {
 		return err
 	}
```
Comment (ymmt2005) [inline clustering/operations.go:168]: ditto

**Review by ymmt2005 — CHANGES_REQUESTED** [review]

File: clustering/operations.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (shunki-fujita) [reply clustering/operations.go:246]: Looping here will prevent updateStatus from being called, so leave the looping to managerProcess.

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Pull request merged**